docs/src/userguide/language_basics.rst

```diff
@@ -178,8 +294,11 @@ Grouping multiple C declarations
 If you have a series of declarations that all begin with :keyword:`cdef`, you
 can group them into a :keyword:`cdef` block like this:
 
+.. note:: This is supported only by Cython language.
```

Comment:
Other option is to use multiple `declare` call it's the option and looks pretty similar

Reply:
`declare()` can be used for multiple variable declaration but the scope of cdef block is more generic (functions, structs etc).

Reply:
multiline* sorry

Reply:
> but the scope of cdef block is more generic (functions, structs etc).

Yeah true, didn't think about it